src/components/USMap/USMap.tsx

```diff
@@ -26,99 +33,155 @@ const stateFipsCodes = Object.keys(statesByFips);
+                  const { stateCode, fullName } = state;
+                  const fillColor = getFillColor(fipsCode);
 
-    const mapContent = useMemo(
-      () => (
-        <USStateMapWrapper $showCounties={showCounties}>
-          <ComposableMap data-tip="" projection={projection} height={500}>
-            <g transform="translate(0, -50)">
-              {showCounties && <MapCounties getFillColor={getFillColor} />}
-              <Geographies geography={stateGeographies}>
-                {({ geographies }) =>
-                  geographies
-                    .filter(geo => stateFipsCodes.includes(geo.id))
-                    .map(geo => {
-                      const fipsCode = geo.id;
-                      // we can coerce this to a state safely because we're only
-                      // dealing with FIPS codes that are from states, because of that filter.
-                      const state = findStateByFipsCodeStrict(
-                        fipsCode,
-                      ) as StateType;
+                  // This flag is used to increase an invisible border around Hawaii and Puerto Rico
+                  // to increase their effective target size
+                  const expandTapArea = ['HI', 'PR'].includes(stateCode);
 
-                      const { stateCode, fullName } = state;
-                      const fillColor = getFillColor(fipsCode);
+                  const geography =
+                    stateCode === 'MP' ? (
+                      <MarianaIslands key={fipsCode} fill={fillColor} />
+                    ) : (
+                      <GeoPath
+                        key={fipsCode}
+                        geography={geo}
+                        fill={fillColor}
+                        fillOpacity={showCounties ? 0 : 1}
+                        stroke="white"
+                        strokeWidth={expandTapArea ? 35 : 1}
+                        strokeOpacity={expandTapArea ? 0 : 1}
+                        role="img"
+                        tabIndex={-1}
+                        aria-label={fullName}
+                      />
+                    );
 
-                      // This flag is used to increase an invisible border around Hawaii and Puerto Rico
-                      // to increase their effective target size
-                      const expandTapArea = ['HI', 'PR'].includes(stateCode);
+                  return (
+                    <USMapTooltip
+                      key={stateCode}
+                      tooltipMode={tooltipMode}
+                      title={renderTooltip(fipsCode, tooltipMode)}
+                      onOpen={() => {
+                        setActiveState(state);
+                      }}
+                      onClose={() => {
+                        setActiveState(activeState =>
+                          activeState === state ? null : activeState,
+                        );
+                      }}
+                      arrow
+                    >
```

Comment:
I feel it looks weird having the tooltip below the cursor, how does it look with `placement="top"`?

Reply:
I'm not sure if I like it better or not (I think either one looks a bit weird; I miss the cursor following behavior).  I'll throw this in a separate PR for you to peek at and weigh in.